core/src/main/scala/kafka/server/LogDirFailureChannel.scala

```diff
@@ -38,7 +41,8 @@ class LogDirFailureChannel(logDirNum: Int) {
    * If the given logDir is not already offline, add it to the
    * set of offline log dirs and enqueue it to the logDirFailureEvent queue
```

Comment:
Note related to this PR: also worth adding a sentence stating that once a directory is offline, it stays that way until a restart.